mkosi.profiles/kiosk/mkosi.conf

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Content]
+Packages=
+        @base-x
+        gdm
+        gnome-kiosk-script-session
+        gnome-text-editor
```

Comment:
The packages are somewhat Fedora-specific, right? This should probably match on the distribution.

Reply:
Yeah, I only tested this on Fedora. Will add a match.